modules/eks-cluster/variables.tf

```diff
@@ -1,4 +1,5 @@
 # ! Developer: if you are adding a variable without a default value, please ensure to reference it in the cleanup script (.github/actions/eks-cleanup-resources/scripts/destroy.sh)
+# and also in the manage gha
```

Comment:
maybe we could go into a bit more detail here what exactly is meant?
Path etc.
`manage gha` = `eks-manage-cluster`?